avm/res/network/bastion-host/main.bicep

```diff
@@ -59,6 +66,14 @@ param tags object?
 @description('Optional. Enable/Disable usage telemetry for module.')
 param enableTelemetry bool = true
 
+@description('Optional. A list of availability zones denoting where the Bastion Host resource needs to come from.')
+@allowed([
+  1
+  2
+  3
+])
+param zones int[] = [] // Availability Zones are currently in preview and only available in certain regions, therefore the default is an empty array.
```

Comment:
We should probably track this in a dedicated issue as there's a spec requiring this to be `[1, 2, 3]`

Reply:
created #3628